### PR TITLE
BAN-2178 Add support for lists to Apple News

### PR DESCRIPTION
**Pull request by damiendillon**
Add support for numbered and unnumbered list items. 
https://mydevex.atlassian.net/browse/BAN-2178

### CODE REVIEW

File: lib/article_json/export/apple_news/elements/list.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class List < Base
+          include ArticleJSON::Export::Common::HTML::Elements::Base
+          include ArticleJSON::Export::Common::HTML::Elements::Text
```
Comment (dsager) [inline lib/article_json/export/apple_news/elements/list.rb]: Are both of these needed to have access to `self.class.exporter_by_type`?

File: lib/article_json/export/apple_news/elements/list.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+          # Each list item should be wrapped in <li></li>
+          #
+          # @return [String]
+          def list
+            @element.content.map do |paragraph_element|
+              line_item = paragraph_element.content.map do |text_element|
+                text_exporter.new(text_element).export
+              end
+              .join
```
Comment (ozgun) [inline lib/article_json/export/apple_news/elements/list.rb]: I would do `end.join` to be consistent with the line 50.

**Comment by ozgun** [comment]
In general it looks good to me, thanks! But, maybe worth to check @dsager 's comment above.

File: lib/article_json/export/apple_news/elements/list.rb (hunk 1 above)
Comment (damiendillon) [reply lib/article_json/export/apple_news/elements/list.rb]: I think they are actually possibly both redundant. Tests pass without them. I will do a manual test to confirm.

Comment (damiendillon) [reply lib/article_json/export/apple_news/elements/list.rb]: Manual test passed. I will remove them now.

File: lib/article_json/export/apple_news/elements/list.rb (hunk 2 above)
Comment (damiendillon) [reply lib/article_json/export/apple_news/elements/list.rb]: Amended

File: lib/article_json/export/apple_news/elements/list.rb (hunk 1 above)
Comment (damiendillon) [reply lib/article_json/export/apple_news/elements/list.rb]: Amended, thanks.

**Pull request merged**